x-pack/test/functional/services/cases/list.ts

```diff
@@ -85,10 +84,13 @@ export function CasesTableServiceProvider(
     },
 
     async validateCasesTableHasNthRows(nrRows: number) {
-      await retry.tryForTime(3000, async () => {
+      await retry.waitFor(`the cases table to have ${nrRows} cases`, async () => {
```

Comment:
Why will `waitFor` be different?

Reply:
Good question! I should put a comment. I noticed that the `waitFor` has bigger waiting times (20sec) and also the ability to provide a message makes it easier to debug.